content/docs/extensions/online_advisor.md

```diff
@@ -0,0 +1,161 @@
+- Activate it by calling any provided function (for example, `get_executor_stats()`)
+
+### Version availability
+
+Please refer to the [Supported Postgres extensions](/docs/extensions/pg-extensions) page for the latest supported version of `online_advisor` on Neon.
+
+## Enable the online_advisor extension
+
+<Admonition type="important">
+Before you can create the `online_advisor` extension **on Neon Postgres 14-16**, you must enable the `online_advisor` preloaded library. See [Extensions with preloaded libraries](/docs/extensions/pg-extensions#extensions-with-preloaded-libraries) for instructions. This step is not required on Postgres 17 projects.
+</Admonition>
+
+You can create the extension in each target database using `CREATE EXTENSION`:
+
+```sql
+CREATE EXTENSION online_advisor;
+```
+
+For information about using the Neon SQL Editor, see [Query with Neon's SQL Editor](/docs/get-started/query-with-neon-sql-editor). For information about using the `psql` client with Neon, see [Connect with psql](/docs/connect/query-with-psql-editor).
+
+## Start collecting recommendations
+
+1. Activate the extension by calling any function:
+
+   ```sql
+   SELECT get_executor_stats();
+   ```
+
+2. Run your workload to collect data.
+
+3. View recommendations:
+
+   ```sql
+   -- Proposed indexes
+   SELECT * FROM proposed_indexes ORDER BY elapsed_sec DESC;
+
+   -- Proposed extended statistics
+   SELECT * FROM proposed_statistics ORDER BY elapsed_sec DESC;
+   ```
+
+## Apply accepted recommendations
+
+Run the `create_index` or `create_statistics` statement from the views, then analyze the table:
+
+```sql
+CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_orders_customer_date ON orders(customer_id, order_date);
+VACUUM (ANALYZE) orders;
+```
+
+## Configure thresholds
+
+You can tune `online_advisor` with these settings for the current session:
+
+| Setting                                  | Default | Description                                                                    |
+| ---------------------------------------- | ------- | ------------------------------------------------------------------------------ |
+| `online_advisor.filtered_threshold`      | `1000`  | Minimum filtered rows in a node to suggest an index.                           |
+| `online_advisor.misestimation_threshold` | `10`    | Minimum actual/estimated row ratio to flag misestimation.                      |
+| `online_advisor.min_rows`                | `1000`  | Minimum returned rows before misestimation is considered.                      |
+| `online_advisor.do_instrumentation`      | `on`    | Toggle data collection.                                                        |
+| `online_advisor.log_duration`            | `off`   | Log planning/execution time for each query.                                    |
+| `online_advisor.prepare_threshold`       | `1.0`   | Planning/execution time ratio above which to suggest prepared statements.      |
+
+<Admonition type="note">
```

Comment:
I'd like thos system level setting to be visble, to know what they are set to but indicate that they cannot be set on Neon. If you need different settings, reach out to Neon Support.